src/biogeophys/SoilTemperatureMod.F90

```diff
@@ -734,11 +735,29 @@ subroutine SoilThermProp (bounds, num_urbanc, filter_urbanc, num_nolakec, filter
                endif
             endif
 
-            ! Thermal conductivity of snow, which from Jordan (1991) pp. 18
+            ! Thermal conductivity of snow
             ! Only examine levels from snl(c)+1 -> 0 where snl(c) < 1
             if (snl(c)+1 < 1 .AND. (j >= snl(c)+1) .AND. (j <= 0)) then  
                bw(c,j) = (h2osoi_ice(c,j)+h2osoi_liq(c,j))/(frac_sno(c)*dz(c,j))
-               thk(c,j) = tkair + (7.75e-5_r8 *bw(c,j) + 1.105e-6_r8*bw(c,j)*bw(c,j))*(tkice-tkair)
+!              TODO slevis: Add namelist option thermal_cond_snow and then
+!                           uncomment relevant lines below
+!              select case (thermal_cond_snow)
+!              case ('Jordan1991')
+!                 thk(c,j) = tkair + (7.75e-5_r8 *bw(c,j) + 1.105e-6_r8*bw(c,j)*bw(c,j))*(tkice-tkair)
+!              case ('Sturm1997')
```

Comment:
@wwieder 
I wanted to make sure I had the default value correct... Are keeping Jordan1991 as default or changing it to Sturm1997?

Reply:
...and maybe the right answer is Jordan1991 for now to get bfb same answers and we change the default later in an answer changing tag.